### ISSUE TITLE
Error when rotating 90° chute under a table saw

### ISSUE THREAD

**Issue by AyeGeeToo** [minetest-mods/hopper#33]
Firstly, not sure if this is an issue with your mod as three mods are involved. It is certainly an issue with the user brain to mouse interface...  hehehe
While rearranging nodes in my world I had a More Blocks Table Saw on top of the Hopper and a Sorter next to the Hopper. Using the Better Screwdriver I rotated the Hopper which crashed Luanti with the following error:

```
AsyncErr: Lua: Runtime error from mod 'hopper' in callback LuaABM::trigger(): /home/lexx/.minetest/mods/hopper/utility.lua:101: attempt to index local 'target_inv_info' (a nil value)
stack traceback:
	/home/lexx/.minetest/mods/hopper/utility.lua:101: in function 'take_item_from'
	/home/lexx/.minetest/mods/hopper/abms.lua:110: in function </home/lexx/.minetest/mods/hopper/abms.lua:88>
```
On a side note is there a way to undo that rotate action so I can get back into the world? I'll start looking into it but I may need some help.

It is a single player world in case that makes a difference.

**Comment by AyeGeeToo** [comment]
Managed to shutdown another world by rotating the Hopper next to a Storage Drawer.  Looks like same error:

```
AsyncErr: Lua: Runtime error from mod 'hopper' in callback LuaABM::trigger(): /home/lexx/.minetest/mods/hopper/utility.lua:101: attempt to index local 'target_inv_info' (a nil value)
stack traceback:
	/home/lexx/.minetest/mods/hopper/utility.lua:101: in function 'take_item_from'
	/home/lexx/.minetest/mods/hopper/abms.lua:110: in function </home/lexx/.minetest/mods/hopper/abms.lua:88>
```

I wanted to open the Hopper but had the Screwdriver selected.....   time for a nap methinks!

**Comment by SmallJoker** [comment]
Can reproduce.
![grafik](https://github.com/user-attachments/assets/3f09a153-2308-4e81-a8d3-8e5c16a42347)

```
ACTION[Server]: singleplayer uses screwdriver:screwdriver, pointing at [node under=31,4,-26 above=32,4,-26]
ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'hopper' in callback LuaABM::trigger(): mods/hopper/utility.lua:101: attempt to index local 'target_inv_info' (a nil value)
ERROR[Main]: stack traceback:
ERROR[Main]: 	mods/hopper/utility.lua:101: in function 'take_item_from'
ERROR[Main]: 	mods/hopper/abms.lua:110: in function 'func'
ERROR[Main]: 	builtin/profiler/instrumentation.lua:124: in function <builtin/profiler/instrumentation.lua:117>
```

**Comment by SmallJoker** [comment]
Fixed as of eee0dc0
